### PR TITLE
dark theme, update context menu, borders, error toast, etc

### PR DESCRIPTION
**Pull request by iqmulator**

### CODE REVIEW

File: src/dotnet/Chat.UI.Blazor/Components/ChatMessageEditor/AttachDropdown.razor [hunk 1]
```diff
@@ -2,14 +2,14 @@
 @using ActualChat.UI.Blazor.Components
 @using ActualChat.Hosting
 
-<Dropdown MenuClass="" ButtonClass="btn-transparent">
+<Dropdown ButtonClass="attach-btn btn-square" Class="flex items-center justify-end">
```
Comment (alexyakunin) [inline src/dotnet/Chat.UI.Blazor/Components/ChatMessageEditor/AttachDropdown.razor:5]: Why `attach-btn`, not `attach-dropdown`?

Comment (alexyakunin) [reply src/dotnet/Chat.UI.Blazor/Components/ChatMessageEditor/AttachDropdown.razor:5]: There is no agreement to add `-btn` or whatever else suffix to the component's main selector.

File: src/dotnet/UI.Blazor/Components/AccountDropdown/AccountMenu.razor [hunk 2]
```diff
@@ -2,12 +2,12 @@
 @using ActualChat.Users
 @using ActualChat.UI.Blazor.Services
 
-<Dropdown Class="relative mt-1" MenuClass="bottom-12 w-48 rounded-md" ButtonClass="!border-0 !pl-0.5 !bg-primary">
+<Dropdown Class="flex items-center justify-end" ButtonClass="account-btn btn-square">
```
Comment (alexyakunin) [inline src/dotnet/UI.Blazor/Components/AccountDropdown/AccountMenu.razor:5]: `account-btn` -> `account-dropdown`?

File: src/nodejs/styles/colors.css [hunk 3]
```diff
@@ -1,26 +1,50 @@
 
 :root, .theme-light {
+    /* MAIN COLORS */
```
Comment (alexyakunin) [inline src/nodejs/styles/colors.css:3]: At which point we started to capitalize comments? :)

**Review by alexyakunin — APPROVED** [review]
IMO makes sense to merge this ASAP - the changes you've made are good, I left a few minor comments.

**Comment by iqmulator** [comment]
toggle button focus
image
right panel button focus
image
input focus, borders, selected text color
image
checkbox focus
image
success button focus (light blue border for light theme)
image
link (a tag) focus
image

no focus for "Hide settings/Show settings" button
no focus for links with class "unfocusable" (chat titles in navbar)

dark theme
image
image
image
image
image
image
image
image
image
image
image
image
image
image
image
image
image
image

**Comment by alexyakunin** [comment]
Looks good - let's merge this & deal with the issues later :)

**Pull request merged**